documentation/book/con-kafka-listeners.adoc

```diff
@@ -108,22 +138,66 @@ listeners:
         nodePort: 32100
       brokers:
       - broker: 0
-        advertisedHost: example.host.1
-        advertisedPort: 12340
         nodePort: 32000
       - broker: 1
-        advertisedHost: example.host.2
-        advertisedPort: 12341
         nodePort: 32001
       - broker: 2
-        advertisedHost: example.host.3
-        advertisedPort: 12342
         nodePort: 32002
 # ...
 ----
 
 For more information on using node ports to access Kafka, see xref:proc-accessing-kafka-using-nodeports-{context}[].
 
+.Customising advertised addresses on external listeners
+
+By default, {ProductName} will try to automatically determine the hostnames and ports which the Kafka cluster should advertise to its clients.
+In some situations this might not be sufficient, because the infrastructure where you run {ProductName} might not provide the right hostname on port under which Kafka can be accessed.
```

Comment:
```suggestion
This is not sufficient in all situations, because the infrastructure on which {ProductName} is running might not provide the right hostname or port through which Kafka can be accessed.
```

Reply:
_Begin a new paragraph._